src/repo/mem.rs

```diff
@@ -107,7 +291,6 @@ impl DataStore for MemDataStore {
     async fn contains(&self, col: Column, key: &[u8]) -> Result<bool, Error> {
         let map = match col {
             Column::Ipns => &self.ipns,
-            Column::Pin => &self.pin,
```

Comment:
removing this without any errors suggests to me that the `Column` object might no longer be needed (in which case this would be a nice `first issue`)? I only see it used in `DataStore` at first glance

Reply:
yes it is internal. I think removal of the whole datastore trait along with this column is inevitable but we'll know more when I bang the file based block/k-v store into shape.